plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actions/Action.java

```diff
@@ -51,6 +62,8 @@ public Action(final String kind) {
 
    public void setReceivedFromClient(final boolean receivedFromClient) { this.receivedFromClient = receivedFromClient; }
 
+   public String getSubclientId() { return subclientId; }
```

Comment:
I think it would be good to already return the FALLBACK_SUBCLIENT_ID here if the id is null.
This would ensure that the subClientId is always defined and we don't run into potential null-pointers in the future.